src/test/java/CardNumberCorrectValidationTest.java

```diff
@@ -0,0 +1,116 @@
+import dev.andrylat.vomelianchuk.CardNumberOperations.CardNumberProcess;
+import dev.andrylat.vomelianchuk.CardNumberOperations.enums.PaymentSystem;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class CardNumberCorrectValidationTest {
+    private final static String descriptionCorrectCardNumber = "\nCard is valid. Payment System is ";
+
+    String expectedResult;
+    String actualResult;
+
+    @Test
+    public void shouldDescriptionVisaPaymentSystem_WhenCardNumberIsCorrectAndStarsWith4() {
+        expectedResult = new CardNumberProcess("4442 1144 6253 6845").operate();
+        actualResult = descriptionCorrectCardNumber + PaymentSystem.VISA;
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void shouldDescriptionMastercardPaymentSystem_WhenCardNumberIsCorrectAndStarsWith51() {
+        expectedResult = new CardNumberProcess("5157 6238 9823 4116").operate();
+        actualResult = descriptionCorrectCardNumber + PaymentSystem.MASTERCARD;
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void shouldDescriptionMastercardPaymentSystem_WhenCardNumberIsCorrectAndStarsWith52() {
+        expectedResult = new CardNumberProcess("5257 6238 9823 4115").operate();
+        actualResult = descriptionCorrectCardNumber + PaymentSystem.MASTERCARD;
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void shouldDescriptionMastercardPaymentSystem_WhenCardNumberIsCorrectAndStarsWith53() {
+        expectedResult = new CardNumberProcess("5357 6238 9823 4114").operate();
+        actualResult = descriptionCorrectCardNumber + PaymentSystem.MASTERCARD;
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void shouldDescriptionMastercardPaymentSystem_WhenCardNumberIsCorrectAndStarsWith54() {
+        expectedResult = new CardNumberProcess("5457 6238 9823 4113").operate();
+        actualResult = descriptionCorrectCardNumber + PaymentSystem.MASTERCARD;
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void shouldDescriptionMastercardPaymentSystem_WhenCardNumberIsCorrectAndStarsWith55() {
+        expectedResult = new CardNumberProcess("5557 6238 9823 4112").operate();
+        actualResult = descriptionCorrectCardNumber + PaymentSystem.MASTERCARD;
+
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void shouldDescriptionDinersClubPaymentSystem_WhenCardNumberIsCorrectAndStarsWith36() {
```

Comment:
You can use parametric tests. In this case it is very good to apply them there. And yes, I know that in the ticket there is another information:) You should not trust tickets 100%.

Reply:
Fixed here: 1783555